back/src/auth/auth.module.ts

```diff
@@ -11,7 +11,7 @@ import { DeviceService } from './device/device.service';
 import { PrismaModule } from '@prisma/prisma.module';
 
 @Module({
-	imports: [UserModule, JwtModule, PrismaModule],
+	imports: [forwardRef(() => UserModule), JwtModule, PrismaModule],
```

Comment:
If you have time, could you explicit this syntax please ?

Reply:
It's only for circular dependencies purpose
As AuthModule import UserModule which import StatusModule which import ChatModule which import AuthModule etc.
The forwardRef just permit to break the circular depency between several modules